src/Entity/Station.php

```diff
@@ -109,6 +109,12 @@ class Station
      */
     private $updatedAt;
 
+    /**
+     * @Gedmo\Timestampable(on="update")
```

Comment:
Update a station must not update lastActivity as we already have updatedAt for this information.